### PR TITLE
Hello Microverse Project pull request

### PR DESCRIPTION
**Pull request by Habtamu210**
HTML and CSS project

### CODE REVIEW

File: A.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: brown;
+}
```
Comment (Santiago220991) [inline A.css:3]: - [OPTIONAL] Please remember that the commit messages should be descriptive, written in the imperative tense, and capitalized. For example in this case could be:  "Add xxxx.css file." Try to improve this in future projects.  :nerd_face:

File: A.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="A.css">
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
+</body>
+</html>
```
Comment (Santiago220991) [inline A.html:13]: - [OPTIONAL] Same here the commit message should be descriptive. For example, Add xxxx.html file. :grin:

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Habtamu210 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Meaningful name branch.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Correct GitHub flow.

## Required Changes ♻️

- Kindly customize the README file using the template provided by Microverse [Link](https://github.com/microverseinc/readme-template). This is a project requirement. :scroll: 
- Kindly edit the Pull Request it should have a meaningful title and a short summary describing the introduced changes. This is also a project requirement. :scroll: 

_Also please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Habtamu210, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

## Required Changes ♻️
- [ ] As suggested by the previous reviewer, I kindly suggest that you add a more descriptive PR title and description. 
## EXAMPLE :
```
## Hello Microverse
In this Pull Request, I:
- Created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Added a HTML file
- Added a CSS File
- Added a h1 text of "Hello-World"
```
https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4

- [ ] As suggested by the previous reviewer, I kindly suggest that you edit your [README](https://github.com/AkashaSedeqi/HelloMicroverse/blob/base-branch/README.md) file to your repo, using the Microverse [template](https://github.com/microverseinc/readme-template) as this will give users detailed information about your project. Here is demo below

https://user-images.githubusercontent.com/52934128/176309876-98f49c4e-3e62-4edd-9bf3-51a66f48e2cd.mp4

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Habtamu210 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Neat Commit messages


 ## Required Changes ♻️

- [ ] Please refer to the previous reviewer comments about a descriptive pull request. He gave you and example that you should follow. Kindly implement it. This is important for your future projects as well. Providing a descriptive title and summary for your PR helps your collaborators know what you did.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Habtamu210** [comment]
HI
"Please refer to the previous reviewer's comments about a descriptive pull
request."
where can I include these comments put in quotations in pull requests?
Thank you!

**Review by Olamarx — COMMENTED** [review]
Hi @Habtamu210, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that can not be reviewed because it has been merged. I can't see the linters and some other features because you have merged it. Kindly always submit for review and make sure it is approved before you merge it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**